src/otx/algo/detection/heads/custom_ssd_head.py

```diff
@@ -33,6 +105,189 @@ def __init__(self, *args, loss_cls: Config | dict | None = None, **kwargs) -> No
             }
         self.loss_cls = MODELS.build(loss_cls)
 
+        self._init_layers()
+
+        bbox_coder.pop("type")
```

Comment:
It seems we don't need to `type` for `bbox_coder` in recipe if using only `DeltaXYWHBBoxCoder`, not popping it here.

Reply:
And when is this file's name changed? We don't need to use `custom_ssd_head` anymore.

Reply:
For the type, we can not remove at this phase, since all the object detector export throught mmdeploy and mmdeploy requires mmconfig with type

Reply:
For export, I understood. It can be updated near future. And what about this file's name?

Reply:
I'll modify file name after this PR for safety